src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer start work");
+    }
+    @Override
+    public void stopWork() {

```

Comment:
The message in the `System.out.println` should be more informative. It should indicate that it's the `Bulldozer` that is stopping its work, in a more descriptive manner.